docker-compose.yaml

```diff
@@ -7,13 +7,13 @@ services:
     volumes:
       - redis_data:/data
 
-  asynqmon:
-    image: hibiken/asynqmon:latest
-    container_name: signer-asynqmon
-    ports:
-      - "${ASYNQMON_PORT:-8020}:8080"
-    environment:
-      REDIS_URL: redis://signer-redis:6379/0
+  # asynqmon:
+  #   image: hibiken/asynqmon:latest
```

Comment:
why comment out this?

Reply:
It is not needed and that docker image not available now.
Let me remove it, rather than commen out.